### PR TITLE
Provide a more practical example regex

### PR DESCRIPTION
**Pull request by cheako**
### Summary
Currently user is restricted to letters e through t and that's a good example, but would almost never exist in the real world.  Thus provided is a regex that might be used to filter usernames.

### Motivation
I think it's less arbitrary and thus not confusing in an off the wall sort of way.

### References
No reply to my inquiry on IRC about this.
https://irclog.perlgeek.de/mojo/2017-04-23#i_14470938

### CODE REVIEW

File: lib/Mojolicious/Guides/Rendering.pod [hunk 1]
```diff
@@ -819,8 +819,8 @@ logic with methods like L<Mojolicious::Validator::Validation/"is_valid">.
     my $validation = $c->validation;
     return $c->render unless $validation->has_data;
 
-    # Validate parameters ("pass_again" depends on "pass")
-    $validation->required('user')->size(1, 20)->like(qr/^[e-t]+$/);
+    # Validate parameters ("user" size is withing 1 - 20 and content looks somewhat like a valid unix username)
```
Comment (jhthorsen) [inline lib/Mojolicious/Guides/Rendering.pod]: How is this related to the proposed change?

Comment (cheako) [inline lib/Mojolicious/Guides/Rendering.pod]: Changed the code and update the comment, silently ignoring that the comment was not altered after a copy/paste.

Comment (cheako) [inline lib/Mojolicious/Guides/Rendering.pod]: s/withing/within/

**Comment by marcusramberg** [comment]
I don't like that this regex is longer and breaks the lines. If the problem is that e-t doesn't constitute a normal username restriction, I think we could change it to [a-z] or \w.

**Comment by kraih** [comment]
Agree with @marcusramberg. https://github.com/kraih/mojo/commit/6b29ec87a5b6ec0730975b000fd341446b23c7c1

**Comment by marcusramberg** [comment]
Closing this as it was obsoleted by sri 10 days ago.
